doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,130 @@
+[[integrating_prometheus_alerts]]
+== OpenShift Prometheus Alerts
+
+[NOTE]
+====
+This feature is available with this release of {product-title} as a technology preview only. For more information on the support scope for features marked as technology previews, see https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope].
+====
+
+Prometheus is used as an external alerting component. {product-title_short} integrates Prometheus alerts for use with OpenShift Container Platform. {product-title_short} collects EmsEvents from Prometheus, generates alerts based on these events, and then attaches alerts to inventory objects.
+
+It is possible to view ongoing alerts in {product-title_short} and manage their life cycle, including the ability to:
+
+* View active alerts per provider in the alert dashboard.
+* View alert data and related objects.
+* Assign and acknowledge alerts in the alerts list screen.
```

Comment:
Comment, assign and acknowledge alerts in the "All Alerts" screen.

Reply:
Updated, thanks.